client/src/pages/createAccount/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.css';
+import { Users } from '../../../../imports/api/users.js';
+
+ export default class CreateAccount extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      firstName:'',
+      surName:'',
+      email:'',
+      password:'',
+      rePassword:''
+    };
+  }
+
+  handleSubmit = () => {
+    const {firstName, lastName, email, password} = this.state;
+    Users.insert({
+      firstname: firstName,
+      surname: lastName,
+      email: email,
+      password: password,
+      createdAt: new Date()
+    });
+  }
+
+  handleFirstNameChange = (event) => {
+    this.setState({firstName: event.target.value});
+  }
+
+  handleSurNameChange = (event) => {
+    this.setState({surName: event.target.value});
+  }
+
+  handleEmailChange = (event) => {
+    this.setState({email: event.target.value});
+  }
+
+  handlePasswordChange = (event) => {
+    this.setState({password: event.target.value});
+  }
+
+  handleRePasswordChange = (event) => {
+    this.setState({rePassword: event.target.value});
+  }
+
+  render() {
+    return (
+      <div className="formWrapper">
```

Comment:
better use some semantic elements, also no need for trailing slashes for self closing elements